lib/lightning_web/live/credential_live/credential_form_component.ex

```diff
@@ -529,10 +538,7 @@ defmodule LightningWeb.CredentialLive.CredentialFormComponent do
                   </div>
                 </fieldset>
               </div>
-              <div
-                :if={@action == :edit and @allow_credential_transfer}
-                class="space-y-4"
-              >
+              <div :if={@action == :edit and true} class="space-y-4">
```

Comment:
elias to change back to `@allow_credential_transfer`

Reply:
Thanks this is done.